### ISSUE TITLE
axis=1 forgotten in tf.reduce_any and tf.reduce_all in TacoTestHelper

### ISSUE THREAD

**Issue by daguix** [Rayhane-mamah/Tacotron-2#216]
Hi,
  For making batch synthesis work, there is a code error, you need to add axis=1 to tf.reduce_any or tf.reduce_all or else it will stop decoding for all sentences in a batch at once if one of them is finished decoding.
Regards.

**Comment by hccho2** [comment]
I agree with daguix.
